docs/common/02_etiquette.md

```diff
@@ -14,3 +16,20 @@ That is why it should only be used for the following simple tasks:
 - Submit jobs
 - Edit scripts and files
 - Prepare / delete data (scp, cp, mv, rm etc.)
+- Run effortless scripts (no computations)
+- Compile software
```

Comment:
I think they should do that in an interactive session.. Compilations can block the head node.

Reply:
I think that it should be allowed for the sake of a smooth workflow, at least for very small applications... Also, I do not know of any system which explicitly forbids or discourages that.

Reply:
what about writing instead `Compile small software packages (e.g., an in-house code).`

Reply:
I would like to have a dedicated section / node for that so `srun -w compile_node --pty /bin/bash` is guaranteed to work.

For example, compiling `gcc` takes over the entire node (@eojons might recall being locked out when I did that) which is an anti-pattern.

Reply:
I agree that it should mention in-house code can be compiled on the login node, but I'd like to encourage and keep a separate node as well.

Reply:
Sounds good... We could remove that line then? I guess that small packages compiling would be innocuous and could be done on login so one does not to have to potentially wait in queue. But I guess no need to be explicit about this...

Thanks!

Reply:
Helloes; yes the idea was indeed to allow for compiling small applications, i.e. people developing software etc. We do not want the common users to be installing their own gcc :P So I like the change to "Compile small software packages (e.g., an in-house code)" and we'll later link to another section where we need to flesh out how that works with Nix and more.